organization/src/main/scala/com/improving/app/organization/Main.scala

```diff
@@ -1,14 +1,65 @@
 package com.improving.app.organization
 
-import com.typesafe.scalalogging.StrictLogging
+import akka.actor.typed.ActorSystem
+import akka.actor.typed.scaladsl.Behaviors
+import akka.stream.alpakka.cassandra.scaladsl.CassandraSessionRegistry
+import com.improving.app.organization.repository._
+import com.typesafe.config.ConfigFactory
+import org.slf4j.LoggerFactory
 
-// This class was initially generated based on the .proto definition by Kalix tooling.
-//
-// As long as this file exists it will not be overwritten: you can maintain it yourself,
-// or delete it so it is regenerated as needed.
+import scala.util.control.NonFatal
 
-object Main extends App with StrictLogging{
+object Main {
 
-    logger.info("starting the Organization service")
+  val logger = LoggerFactory.getLogger("organization.Main")
+
+  def main(args: Array[String]): Unit = {
+    val conf = ConfigFactory
+      .parseString("akka.http.server.preview.enable-http2 = on")
+      .withFallback(ConfigFactory.defaultApplication())
+    val system =
+      ActorSystem[Nothing](
+        Behaviors.empty,
+        "OrganizationService",
+        conf.resolve()
+      )
+    try {
+      init(system)
+    } catch {
+      case NonFatal(e) =>
+        logger.error("Terminating due to initialization failure.", e)
+        system.terminate()
+    }
+  }
+
+  def init(implicit system: ActorSystem[_]): Unit = {
+
+    val session = CassandraSessionRegistry(system).sessionFor(
+      "akka.persistence.cassandra"
+    )
+    // use same keyspace for the item_popularity table as the offset store
```

Comment:
This comment seems like it comes from the framework you adapted from to write this file. Please either take it out or make it up to date with your adaptations.